test/fixtures/valid-es5.js

```diff
@@ -234,10 +234,37 @@
 	 * @param {RegExp} j
 	 * @return {Error}
 	*/
-	APP.typing = function ( a, b, c, d, e, f, g, h, i, j ) {
+	APP.JSDocTypes = function ( a, b, c, d, e, f, g, h, i, j ) {
 		return a || b || c || d || e || f || g || h || i || j;
 	};
 
+	/**
+	 * {@link APP}
+	 * {@link APP}
+	 *
+	 * @abstract
+	 * @extends String
+	 * @class
+	 * @constructor
+	 * @constructs
+	 * @const
+	 * @default
+	 * @desc Description.
+	 * @external String
+	 * @file Description.
+	 * @emits Event
```

Comment:
According to Codesearch, there are no uses of `@emits` in WMF-deployed repo. Instead, we use `@fires`.

Reply:
:+1: done.